### DISCUSSION TITLE
app close every time i try to convert to cxdlp

### DISCUSSION THREAD

**Discussion by felipeyts** [sn4k3/UVtools#466] [General]
tried several times with several files and program stops to work while converting to creality cxdlp file
anyone have any other way to convert cxdlp files?

**Comment by sn4k3** [comment]
Printscreen the about box and paste here.
Also send the target file via link.

Does it convert to any other format?

**Comment by felipeyts** [comment]
The dialog box is always empty and program shut down, did not tried any other format as my printer accept only cxdlp
![crash](https://user-images.githubusercontent.com/104234669/164816785-a8653c7d-698f-42bf-9551-3e946cc1a94c.jpg)

**Reply by sn4k3** [reply]
Not the dialog box, the about box (F1 - Help - about)
Also send the error logs: Help - Open setting folder
